### PR TITLE
Add a REVAULTD_PATH environment variable

### LINKED ISSUE #18: Have a REVAULTD_PATH env var
To set the path to the `revaultd` binary, it's not practical to have to install it globally every time...

### PR DESCRIPTION
**Pull request by danielabrozzoni**
Add an environment variable for specifying revaultd path. When revaultd
isn't running, uses this variable to start the binary. If not provided,
the gui will call the `revaultd` command.

Fixes #18

### CODE REVIEW

File: src/revaultd/mod.rs [hunk 1]
```diff
@@ -178,7 +178,14 @@ pub struct ListOnchainTransactionsResponse {
 // RevaultD can start only if a config path is given.
 pub async fn start_daemon(config_path: &Path) -> Result<(), RevaultDError> {
     debug!("starting revaultd daemon");
-    let child = Command::new("revaultd")
+    let mut command = std::path::PathBuf::new();
+    if let Ok(revaultd_path) = std::env::var("REVAULTD_PATH") {
```
Comment (edouardparis) [inline src/revaultd/mod.rs:182]: We try to keep every ENV VAR parsing in main.rs
https://github.com/edouardparis/revault-gui/blob/fix-ci-workflow/src/main.rs#L14
Rename var for REVAULTD_BIN_PATH and store result in app::Config
https://github.com/edouardparis/revault-gui/blob/fix-ci-workflow/src/ui/app.rs#L122
then pass it here: https://github.com/edouardparis/revault-gui/blob/fix-ci-workflow/src/ui/state/charging.rs#L66

Comment (danielabrozzoni) [reply src/revaultd/mod.rs]: Fixed, thanks :)

**Comment by edouardparis** [comment]
very good, please add new var to [README section](https://github.com/revault/revault-gui/#env-vars)

File: README.md [hunk 2]
```diff
@@ -16,3 +16,4 @@ default `revaultd` configuration location is checked.
 | `REVAULTD_CONF`       | Path to the [revaultd](https://github.com/re-vault/revaultd) configuration path                                                                                          |
 | `REVAULTGUI_DEBUG`    | If `true`, the interface will use `iced` debug feature to display current layout and set log level to `debug`                                                            |
 | `REVAULTGUI_LOG`      | Enable the [tracing env filter](https://docs.rs/tracing-subscriber/0.2.15/tracing_subscriber/filter/struct.EnvFilter.html) example: `revault_gui::revault::client=debug` |
+| `REVAULTD_PATH`       | Path to the [revaultd](https://github.com/re-vault/revaultd) binary                                                                                                      |
```
Comment (edouardparis) [inline README.md:19]: Sorry https://github.com/revault/revaultd

**Review by edouardparis — APPROVED** [review]
ACK 2d48e4c1501a2f4ab81fab9f1b201d31e92ab0e9

**Pull request merged**